vere/main.c

```diff
@@ -388,8 +388,8 @@ u3_ve_usage(c3_i argc, c3_c** argv)
     "   https://github.com/urbit/urbit/blob/master/CONTRIBUTING.md\n",
     "\n",
     "Simple Usage: \n",
-    "   %s -c <mycomet> to create a comet (anonymous urbit)\n",
-    "   %s -w <myplanet> -t <myticket> if you have a ticket\n",
+    "   %s -c <my-comet> to create a comet (anonymous urbit)\n",
+    "   %s -w <my-planet> -k <my-key-file> if you own a planet\n",
```

Comment:
`if you own a planet or other Azimuth point`

Reply:
See also `<my-planet>`, "restart your planet/comet" ...

Reply:
I don't understand what you mean

Reply:
I see. We can leave it as planet, then